gensim/models/doc2vec.py

```diff
@@ -580,7 +580,7 @@ def __init__(self, documents=None, dm_mean=None,
         need about 1GB of RAM. Set to `None` for no limit (default).
 
         `sample` = threshold for configuring which higher-frequency words are randomly downsampled;
-                default is 1e-3, useful value is 1e-5.
+                default is 1e-3, values to 1e-5 (or lower) may also be useful.
```

Comment:
Going back to word2vec.c, `sample=0` meant to turn it off. Values 1.0 and higher were redundant (all having the same 'turn-it-off' meaning). So a convenience hack was added that if you wanted to start downsampling not at some hard-to-calculate tiny float, but a specific word count, you could specify an int `sample`, >=1, and the model would adjust the internal formula so that downsampling would start at that word-frequency. I used it for some experiments, and it might be more intuitive to tune than the float-value, but it's not documented beyond the inline comment. 

IIRC FastText then adopted the convention that `sample=1.0` means off (which has a certain consistency with the downsampling algorithm), but creates the potential for confusion at this boundary number. 

Everything "works" for one purpose or another but might not be clear. 

Really this bit of doc-comment should go into word2vec.py - where sample is defined. (Here, it's merely inherited – and for those values maybe the Doc2Vec comment should just say 'see Word2Vec'.) And, any guidance here should be kept in sync with the `add_parameter` doc-text for command-line invocation, defined at the bottom of word2vec.py.